README.md

```diff
@@ -1,70 +1,176 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
- [ ] When I click the "Rooms" nav link, I get this:

![image](https://github.com/huzaifakhan0308/full-stack-capstone-front-end/assets/57421823/90cc6a7e-6f22-4a1a-b60f-9f93ad72b6ba)

This component seems to be lacking implementation:

![image](https://github.com/huzaifakhan0308/full-stack-capstone-front-end/assets/57421823/839e8383-8c32-4973-9fc3-0da64c68870c)

Kindly, either implement it or remove the link.  That way, the UX will be improved. 👍